clouddriver-kubernetes/src/integration/resources/clouddriver.yml

```diff
@@ -28,6 +29,7 @@ kubernetes:
       metrics: false
       kubeconfigFile: ${PROJECT_ROOT}/clouddriver-kubernetes/build/kubeconfigs/kubecfg-${account1_containername}.yml  # File is automatically created during initialization
     - name: account2
+      cacheIntervalSeconds: 30
```

Comment:
What was the rationale behind having different `cacheIntervalSeconds` for account1 vs account2?

Reply:
I reduced the interval for `account1` just so that it runs the cache refresh more frequently to speed up tests. `account2` is actually not directly used, it's just there so that there is more than one account present, but it could be deleted and added later when there are tests actually needing it.